.github/actions/build-electron/action.yml

```diff
@@ -60,8 +60,7 @@ runs:
           sudo launchctl limit maxfiles 65536 200000
         fi
 
-        NINJA_SUMMARIZE_BUILD=1 e build
-        cp out/Default/.ninja_log out/electron_ninja_log
```

Comment:
Can we not? The `electron_ninja_log` is an artifact for debugging stuff like build perf